Source/View Controller/ProfileViewController/ProfileImageViewControllerExtension.swift

```diff
@@ -19,4 +38,17 @@ extension ProfileViewController: UIImagePickerControllerDelegate, UINavigationCo
     }
     
     // MARK: - Camera Picker Delegate functions
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        if let imageSelected = info[UIImagePickerController.InfoKey.originalImage] as? UIImage {
+            profilePictureButton.imageView?.image = imageSelected
```

Comment:
This wouldn't work for UIButtons, you have to use `profilePictureButton.setImage(imageSelected, for: .normal)`